content/applications/finance/fiscal_localizations/italy.rst

```diff
@@ -122,314 +193,349 @@ Accounting.
-the EU to be used as reference (`0% EU`, invoice label `00eu`), which can be found under
-:menuselection:`Accounting --> Configuration --> Taxes`. Exports are exempt from VAT, and therefore
-**reverse charge** taxes require the :guilabel:`Has exoneration of tax (Italy)` option ticked, with
-both the :guilabel:`Exoneration` kind and :guilabel:`Law Reference` filled in.
+Odoo generates the required XML files as attachments to invoices in the `FatturaPA` format
+required by the :abbr:`AdE (Agenzia delle Entrate)`. Select the invoices and mark
+:guilabel:`Generate XML File` in the :guilabel:`Send and Print` dialog to generate the attachments.
 
-.. image:: italy/italy-tax.png
-   :align: center
-   :alt: External reverse charge settings
+.. image:: italy/italy-edi-menu.png
+   :alt: Send and Print menu
 
-.. note::
-   If you need to use a different kind of :guilabel:`Exoneration`, click :menuselection:`Action -->
-   Duplicate` within the tax menu to create a copy of an existing similar tax. Then, select another
-   :guilabel:`Exoneration`, and :guilabel:`Save`. Repeat this process as many times as you need
-   different kind of :guilabel:`Exoneration` taxes.
+.. image:: italy/italy-edi-send-and-print.png
+   :alt: Send and Print dialog
 
-.. tip::
-   **Rename** your taxes in the :guilabel:`Name` field according to their :guilabel:`Exoneration` to
-   differentiate them easily.
+.. image:: italy/italy-edi-attachments.png
+   :alt: EDI Attachments
+
+Submission to SDI
+~~~~~~~~~~~~~~~~~
+
+Invoices are sent to the :guilabel:`Proxy Server`, which gathers all requests and then forwards them
+via a WebServices channel to the :abbr:`SdI (Sistema di Interscambio)`. Check the sending status of
+the invoice through the :guilabel:`Check Sending` button at the top of the invoice's view.
 
-On your invoice, select the corresponding tax you need in the :guilabel:`Taxes` field. You can find
-the following **additional info** by opening the **XML** file of the issued invoice:
+Processing by SDI
+~~~~~~~~~~~~~~~~~
 
-- :guilabel:`SdI address (Codice Destinatario)`: must be filled for both **EU** or **non-EU**;
-- :guilabel:`Country Id`: must contain the country of the foreign seller in the two-letter ISO
-  (Alpha-2) code (ex., `IT` for 'Italy');
-- :guilabel:`CAP`: must be filled with `00000`;
-- :guilabel:`Partita Iva` (**VAT number**): must contain the **VAT** number for **EU businesses**
-  and `OO99999999999` (double 'O' **letter**, not 'zero') for **non-EU businesses**. In case of
-  private customers without **VAT** number, use `0000000`;
-- :guilabel:`Fiscal Code`: for foreign entities without an actual **Codice Fiscale**, any
-  recognizable identifier is valid.
+The :abbr:`SdI (Sistema di Interscambio)` receives the document and verifies for any errors. At this
+stage, the invoice is in the :guilabel:`SdI Processing` state, visible on the invoice's view.
```

Comment:
stage, the invoice is in the :guilabel:`SdI Processing` state, as shown on the invoice.